src/questions.js

```diff
@@ -0,0 +1,87 @@
+const map = require('lodash.map');
+const longest = require('longest');
+const rightPad = require('right-pad');
+const types = require('conventional-commit-types').types;
+
+function formatTypes(types) {
+  const length = longest(Object.keys(types)).length + 1;
+  return (choices = map(types, function(type, key) {
+    return {
+      name: rightPad(key + ':', length) + ' ' + type.description,
+      value: key,
+    };
+  }));
+}
+
+module.exports = [
+  {
+    type: 'list',
+    name: 'type',
+    message: "Select the type of change that you're committing:",
+    choices: formatTypes(types),
+  },
+  {
+    type: 'input',
+    name: 'scope',
+    message: 'What is the scope of this change? (press enter to skip)',
+  },
+  {
+    type: 'input',
+    name: 'subject',
+    message: 'Write a short, imperative tense description of the change:',
+  },
+  {
+    type: 'input',
+    name: 'body',
+    message:
+      'Provide a longer description of the change: (press enter to skip)\n',
+  },
+  {
+    type: 'confirm',
+    name: 'isBreaking',
+    message: 'Are there any breaking changes?',
+    default: false,
+  },
+  {
+    type: 'input',
+    name: 'breaking',
+    message: 'Describe the breaking changes:\n',
+    when: answers => answers.isBreaking,
+  },
+  {
+    type: 'confirm',
+    name: 'isIssueAffected',
+    message: 'Does this change affect any open Github issues?',
+    default: false,
+  },
+  {
+    type: 'input',
+    name: 'issues',
+    message: 'Add issue references (e.g. "fix #123", "re #123".):',
+    when: answers => answers.isIssueAffected,
+  },
+  {
+    type: 'confirm',
+    name: 'isClubhouseBranch',
+    message: `Would you like to link this branch to any Clubhouse.io stories?`,
```

Comment:
1. Maybe drop the "Would you like to" part for terseness.
2. For even more terseness, "stories" is a bit redundant since it's used in the follow-up question anyway:

-> "Link this commit to Clubhouse?"
-> "What story id(s)? (comma separated)"

-> "Link this branch to Clubhouse?"
-> "What story id(s)? (comma separated)"